content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,494 @@
+         <td>Device Location, is name-value pair having component type and component name</td>
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>Total usable capacity of the array</td>
+      </tr>
+      <tr>
+         <td>Used Capacity</td>
+         <td>Total used capacity</td>
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>Total free capacity</td>
+      </tr>
+      <tr>
+         <td>Raw Capacity</td>
+         <td>Total raw capacity(sum of all disk capacity)</td>
+      </tr>
+      <tr>
+         <td>Subscribed Capacity</td>
+         <td>Total amount of capacity subscribed</td>
+      </tr>
+   </tbody>
+</table>
+ 
+### Alert data matrix
+ 
+<table>
+   <thead>
+      <tr>
+         <th>Attribute</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Alert id</td>
+         <td>Identification of alarm.</td>
+      </tr>
+      <tr>
+         <td>Alert name</td>
+         <td>Name of the alarm</td>
+      </tr>
+      <tr>
+         <td>Severity</td>
+         <td>Severity of the alarm</td>
+      </tr>
+      <tr>
+         <td>Category</td>
+         <td>Category of the reported alarm notification. Ex: Fault, Event or Recovery</td>
+      </tr>
+      <tr>
+         <td>Type</td>
+         <td>Basic classification of the alarm. Ex:CommunicationsAlarm, EquipmentAlarm</td>
+      </tr>
+      <tr>
+         <td>Sequence number</td>
+         <td>Unique identification for the alert instance</td>
+      </tr>
+      <tr>
+         <td>Occur time</td>
+         <td>Time at which alert is generated from device. Specified in epoch format</td>
+      </tr>
+      <tr>
+         <td>Description</td>
+         <td>Alarm Description</td>
+      </tr>
+      <tr>
+         <td>Recovery advice</td>
+         <td>This field does not contain any info. However description field may contain some recovery advice in some cases.</td>
+      </tr>
+      <tr>
+         <td>Resource type</td>
+         <td>Resource type for alert which can be storage subsystem or some other subsystems</td>
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td>Detailed info about tracing the alerting device such as array id, component name, component type</td>
+      </tr>
+   </tbody>
+</table>
+ 
+### Performance metric data
+ 
+<table>
+   <thead>
+      <tr>
+         <th>Resource</th>
+         <th>Metric</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td rowspan=6>Storage Array</td>
+         <td>Bandwidth</td>
+         <td>Total bandwidth in MB/s</td>
+      </tr>
+      <tr>
+         <td>Response time</td>
+         <td>Total read and write response time in MB/s</td>
+      </tr>
+      <tr>
+         <td>Read Bandwidth</td>
+         <td></td>
+      </tr>
+      <tr>
+         <td>Write Bandwidth</td>
+         <td></td>
+      </tr>
+      <tr>
+         <td>Read throughput</td>
```

Comment:
Give only 2 or 3 metrics in each resource.. Give description for all.. Do not leave blank